src/common/options.cc

```diff
@@ -9008,7 +9008,7 @@ std::vector<Option> get_mds_client_options() {
     .set_default(30)
     .set_description(""),
 
-    Option("client_caps_release_delay", Option::TYPE_INT, Option::LEVEL_DEV)
+    Option("client_caps_release_delay", Option::TYPE_SECS, Option::LEVEL_DEV)
```

Comment:
When you're cherry-picking the commit and this file should conflicted, so in commit comment you should add something like:

```
    Conflicts:
            src/common/options.cc
```

Reply:
Added the conflict files with explanation in commit desc.